src/nonfungiblePositionManager.ts

```diff
@@ -74,6 +75,27 @@ export type IncreaseOptions = CommonAddLiquidityOptions & IncreaseSpecificOption
 
 export type AddLiquidityOptions = MintOptions | IncreaseOptions
 
+export interface SafeTransferOptions {
+  /**
+   * The account sending tne NFT.
+   */
+  sender: string
+
+  /**
+   * The account that should receive the NFT.
+   */
+  recipient: string
+
+  /**
+   * The id of the token being sent.
+   */
+  tokenId: BigintIsh
+  /**
+   * The optional parameter that passes data to the `onERC721Received` call for the staker
+   */
+  data?: IncentiveKey | IncentiveKey[]
```

Comment:
data should be a hex string here, the nonfungible position manager can be transferred to any number of contracts that can handle them, not just the staker